src/rollbar.js

```diff
@@ -42,6 +43,7 @@ Rollbar.prototype.configure = function(options, payloadData) {
     payload = {payload: payloadData};
   }
   this.options = _.extend(true, {}, oldOptions, options, payload);
+  this.global(this.options);
```

Comment:
This allows global settings to come through configure which I suspect people are also doing.